src/styles/_variables.scss

```diff
@@ -0,0 +1,3 @@
+$color-word: #060b35;
+$color-word2: #616070;
+$color-botton: #00acdc;
```

Comment:
There is a typo in the variable name. Change $color-botton to $color-button for better readability and consistency.